src/Capability/Registry/Loader/RegistryLoaderInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the official PHP MCP SDK.
+ *
+ * A collaboration between Symfony and the PHP Foundation.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Mcp\Capability\Registry\Loader;
+
+use Mcp\Capability\Registry\ReferenceRegistryInterface;
+
+interface RegistryLoaderInterface
```

Comment:
I feel like we can drop the `Registry` in interface and class names since it's already part of the namespace